### PR TITLE
Small energy pulse rifle update

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Updates the energy pulse rifle from #3124 as I don't have time left to update the weapon before it got merged. Adds a new 'pulse' image, new pulse magazine sprite and new sounds in general.

## Why It's Good For The Game
Because UPDATES, son.

## Changelog
:cl:
tweak: Updated the energy pulse rifle because I didn't have time to merge that carp.
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
I'll get the conflicts done by tomorrow, gotta heckin' sleep.

File: code/modules/projectiles/updated_projectiles/magazines/energy.dm [hunk 1]
```diff
@@ -1,49 +1,57 @@
...
-	base_ammo_icon = "m43"
-	icon_state = "m43"
-	gun_type = /obj/item/weapon/gun/energy/lasgun/pulse
-	maxcharge = 2000 // 100 shots.
-	self_recharge = TRUE
-	charge_amount = 25 // 10%, 1 shot
-	charge_delay = 2 SECONDS
+//Energy weapons
+
+/obj/item/cell/lasgun
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/magazines/energy.dm]: End of line changes.

Comment (XSlayer300) [reply code/modules/projectiles/updated_projectiles/magazines/energy.dm]: Thanks.

Comment (Rohesie) [reply code/modules/projectiles/updated_projectiles/magazines/energy.dm]: These are still a thing.

Comment (XSlayer300) [reply code/modules/projectiles/updated_projectiles/magazines/energy.dm]: Oh really? Carp.

File: code/modules/projectiles/updated_projectiles/magazines/energy.dm [hunk 2]
```diff
@@ -1,49 +1,56 @@
...
+	name = "\improper Lasgun Battery"
+	desc = "A specialized high density battery used to power Lasguns."
+	icon = 'icons/obj/items/ammo.dmi'
+	icon_state = "m43"
+	maxcharge = 1000
+	w_class = WEIGHT_CLASS_NORMAL
+	var/base_ammo_icon = "m43"
+	var/gun_type = /obj/item/weapon/gun/energy/lasgun
+	var/reload_delay = 0
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/magazines/energy.dm]: yep still EOL changes

File: code/modules/projectiles/updated_projectiles/magazines/energy.dm (hunk 1 above)
Comment (spookydonut) [reply code/modules/projectiles/updated_projectiles/magazines/energy.dm]: this is actually changing it from CRLF to LF, its fine

**Pull request merged**